packages/storage/storage.type.ts

```diff
@@ -51,9 +51,15 @@ export interface SimpleStorage<T extends MarshallingType> {
   get(): T | null;
   /**
    * 스토리지에 값을 설정한다.
-   * @param {T} value
+   * @param value 설정할 값
    */
   set(value: T): void;
+  /**
+   * 스토리지에 값을 설정한다.
+   * @param value 설정할 값
+   * @param expiredDate 다시 설정 할 만료일자
+   */
+  set(value: T, expiredDate: string): void;
```

Comment:
set() 메서드를 사용하는 위치에서는 expiredDate를 undefined or string으로 판단 중이니 해당 interface 안에 set() 메서드에 대한 타입은 하나만 존재해도 되지 않을까 싶습니다! 혹시 이렇게 하신 이유가 있으실까요~?

```suggestion
  set(value: T, expiredDate?: string): void;
```

Reply:
아, 이 부분은 다른 개발자분들이 사용 시 `overloading` 메서드로 사용토록 유도하기 위함이었습니다!
그래서 말씀하신 단일 value 만, 그리고 value 및 expiredDate 2가지를 동시에 원하는 메서드 2가지를 인터페이스상에 아래와 같이 함께 선언 해 두었습니다!

```ts
  /**
   * 스토리지에 값을 설정한다.
   * @param value 설정할 값
   */
  set(value: T): void;
  /**
   * 스토리지에 값을 설정한다.
   * @param value 설정할 값
   * @param expiredDate 다시 설정 할 만료일자
   */
  set(value: T, expiredDate: string): void;
```
물론 실제 구현단계에서는 두번째 인자를 optional 로 두어 타입 체크하여 사용하고 있습니다~

인터페이스 메서드 내 주석이 길어서 못보신게 아닐까... 하는데 혹시 제 답변이 적절치 않다면 다시 피드백 부탁드리겠습니다~ 🙇‍♂️